website/content/api-docs/system/remount.mdx

```diff
@@ -27,7 +43,16 @@ This endpoint moves an already-mounted backend to a new mount point.
 
 - `to` `(string: <required>)` – Specifies the new destination mount point.
 
-### Sample Payload
+### Sample Payload ( Cross Namespace)
+
+```json
+{
+  "from": "ns1/ns2/secret",
```

Comment:
I think it would be good to add an example for auth as well since we need to prefix "from" and "to" fields with auth/. Would be good to see how the field values look with namespaces.

Reply:
added an example [ec185fc](https://github.com/hashicorp/vault/pull/14314/commits/ec185fca4801fb73794fecd7dda77eea8a8e6d22)